### PR TITLE
some grammeritical changes

### PR DESCRIPTION
**Pull request by ravikr126**
### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Publicizing or hiding organization membership
-intro: 'If you''d like to tell the world which organizations you belong to, you can display the avatars of the organizations on your profile.'
+intro: 'If you''d like to show the world which organizations you belong to, you can show your avatars of the organizations on your profile.'
```
Comment (Rexogamer) [inline content/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership.md:3]:
```suggestion
intro: 'If you''d like to show the world which organizations you belong to, you can show the avatars of these organizations on your profile.'
```

(I'm a bit unsure about the `you can show the avatars of these organizations` bit. Any suggestions?)

**Comment by ravikr126** [comment]
Grammatical changes

**Review by SMAKSS — COMMENTED** [review]
Hi there, thanks for the PR. I think your changes are still subjective and there is no improvement whatsoever.

**Comment by Rexogamer** [comment]
> Hi there, thanks for the PR. I think your changes are still subjective and there is no improvement whatsoever.

Whilst this likely isn't your intention, that second sentence comes off as a little bit rude :/

**Comment by ravikr126** [comment]
> > Hi there, thanks for the PR. I think your changes are still subjective and there is no improvement whatsoever.
> 
> Whilst this likely isn't your intention, that second sentence comes off as a little bit rude :/

How I make contribution in open source ??
Please help me

**Comment by SMAKSS** [comment]
> > Hi there, thanks for the PR. I think your changes are still subjective and there is no improvement whatsoever.
> 
> Whilst this likely isn't your intention, that second sentence comes off as a little bit rude :/

Well, it may [sound too hostile](https://meta.stackexchange.com/questions/242159/the-new-edit-rejection-message-for-no-improvement-whatsoever-sounds-too-hostil) but it is actually not.

**Comment by SMAKSS** [comment]
> > > Hi there, thanks for the PR. I think your changes are still subjective and there is no improvement whatsoever.
> > 
> > 
> > Whilst this likely isn't your intention, that second sentence comes off as a little bit rude :/
> 
> How I make contribution in open source ??
> Please help me

If you want to contribute to the open-source repos, first of all, you have to carefully read the [contribution section](https://github.com/github/docs#contributing) of that repo, then you can keep an eye on the [issue section](https://github.com/github/docs/issues), once you found one suitable for you, you can fix it *(or ask the owners for working on that issue)* then you can apply a proper PR.

This is how it usually works, but you can sometimes go the other way around (Find the Issue yourself post the issue and then fix it with a PR).

**Comment by ravikr126** [comment]
Thanks brother😊

**Comment by janiceilene** [comment]
@ravikr126 Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

I agree that these changes are subjective, but if there's a reason you'd like to see those changes, we can consider them. By filling out the pull request template, the team can better understand the goal of your changes when we review your PR 💖

**Comment by janiceilene** [comment]
@ravikr126 Could you Tplease fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, if you'd like this change to be considered?

**Review by tashaemery — APPROVED** [review]

**Review by tashaemery — APPROVED** [review]

**Review by ravikr126 — COMMENTED** [review]
grammatical enhancement

**Comment by janiceilene** [comment]
@ravikr126 Could you answer the **Why:** and **What's being changed:** in the top comment to help us understand the reason you feel these changes should be made? Especially since this change is subjective, it's helpful to know your thoughts behind the change so we can decide whether or not to merge it into the docs.

**Pull request closed without merging**